qCC/db_tree/ccDBRoot.cpp

```diff
@@ -1700,6 +1704,7 @@ void ccDBRoot::selectByTypeAndName()
 	//For generic-only types the match type gets overridden and forced to
 	//false because exclusive match makes no sense!
 	bool exclusive;
+    bool regex;
```

Comment:
bool regex is initialized, as well the exclusive var, into the default case next lines. but maybe is better to get it out of it...